quizbizz/src/App.jsx

```diff
@@ -25,6 +25,7 @@ function App() {
             </RequiredAuth>
           }
         />
+        <Route path="/quizquestion" element={<QuizQuestion />}></Route>
```

Comment:
Why you have not wrapped this file with RequiredAuth ??

Reply:
Done